k8-util/helm/fluvio-sys/templates/crd_managed_connector.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
bump up chart version

Reply:
To which part? I tried a newer `apiVersion` and it doesn't work on my system.

Reply:
https://github.com/infinyon/fluvio/blob/master/k8-util/helm/fluvio-sys/Chart.yaml.

version field